<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>influenceModeQWidget</class>
 <widget class="QWidget" name="influenceModeQWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="influenceModeQGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Influence Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="influenceModeQLayout">
        <item>
         <widget class="QLabel" name="influenceModeQLabel">
          <property name="text">
           <string>Mesh:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="influenceModeQLine">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Set mesh selected to use it</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="influenceModeQButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="influenceModeQTree">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Influences</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="influenceProxyQGroup">
        <property name="title">
         <string>Influence Proxy Action:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QRadioButton" name="influenceProxyReplaceQRadio">
           <property name="text">
            <string>Replace</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="influenceProxyAddQRadio">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="influenceVertexAreaQCheck">
        <property name="text">
         <string>List Vertex Influence Area</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="influenceVertexAreaQLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
